src/devtools/utils.ts

```diff
@@ -77,12 +105,12 @@ export function useIsMounted() {
  * to prevent updating a component state while React is rendering different components
  * or when the component is not mounted anymore.
  */
-export function useSafeState(initialState) {
+export function useSafeState<T>(initialState: T): [T, (value: T) => void] {
   const isMounted = useIsMounted()
   const [state, setState] = React.useState(initialState)
 
   const safeSetState = React.useCallback(
-    value => {
+    (value: T) => {
```

Comment:
getting now:

> React Hook React.useCallback has a missing dependency: 'T'. Either include it or remove the dependency array

I think this can potentially be solved with updating the `eslint-plugin-react-hooks` to the latest version

Reply:
Fixed ✅